.gitignore

```diff
@@ -30,3 +30,6 @@ yarn.lock*
 package-lock.json
 
 *.zip
+
+# ide specific
+.idea
```

Comment:
nit: i put stuff like this in my gitignore global file

Reply:
I should also do that, but also it's nice for others committing to not have to worry about it :)